fed/mosaic-environment/src/main/java/org/eclipse/mosaic/fed/environment/config/CEventLocation.java

```diff
@@ -36,7 +36,7 @@
     public GeoArea area;
 
     /**
-     * The ID of the road segment the event is located on (Connection ID or road ID).
+     * The ID of the road segment the event is located on (Connection ID).
      */
     public String roadSegmentId;
```

Comment:
Why not rename it to `connectionId`?

Reply:
I guess for the sake of compatibility, but I could rename this and use `@SerializedName` to allow alternative names.